develop/dev-guide-gui-dbeaver.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Connect to TiDB with DBeaver
+summary: Learn how to connect to TiDB using DBeaver Community.
+---
+
+# Connect to TiDB with DBeaver
+
+TiDB is a MySQL-compatible database, and [DBeaver Community](https://dbeaver.io/download/) is a free cross-platform database tool for developers, database administrators, analysts, and everyone working with data.
+
+In this tutorial, you can learn how to connect to your TiDB cluster using DBeaver Community.
+
+> **Note:**
+>
+> This tutorial is compatible with TiDB Serverless, TiDB Dedicated, and TiDB Self-Hosted.
+
+## Prerequisites
+
+To complete this tutorial, you need:
+
+- [DBeaver Community **23.0.3** or higher](https://dbeaver.io/download/).
```

Comment:
The latest tested version listed in `dev-guide-third-party-support.md` is 23.0.1. Should we update this to match the latest tested version?
https://github.com/pingcap/docs/blob/88124e9776b11e2c1799b2d9a5cfa08c0a7ea517/develop/dev-guide-third-party-support.md?plain=1#L182

Reply:
good catch!